test/cli/kit/system_test.rb

```diff
@@ -65,8 +80,11 @@ def test_system_finds_ruby_script_with_path_modifications
 
       def with_script_in_tmpdir(script_name)
         Dir.mktmpdir do |tmpdir|
+          is_windows = CLI::Kit::System.os == :windows
+          script_name += '.BAT' if is_windows
+
           File.open(File.join(tmpdir, script_name), "w", 0755) do |f|
-            f.write("#!/bin/sh\n")
+            f.write("#!/bin/sh\n") unless is_windows
```

Comment:
...prevent the command output?
```suggestion
            if is_windows
              f.write("@echo off\n")
            else
              f.write("#!/bin/sh\n")
            end
```